backends/advanced/init.py

```
@@ -566,6 +566,72 @@ def setup_knowledge_graph(self):
+        # Check if keys were passed from wizard (langfuse init already ran)
+        langfuse_pub = getattr(self.args, 'langfuse_public_key', None)
+        langfuse_sec = getattr(self.args, 'langfuse_secret_key', None)
+
+        if langfuse_pub and langfuse_sec:
+            # Auto-configure from wizard — no prompts needed
+            self.config["LANGFUSE_HOST"] = "http://langfuse-web:3000"
+            self.config["LANGFUSE_PUBLIC_KEY"] = langfuse_pub
+            self.config["LANGFUSE_SECRET_KEY"] = langfuse_sec
+            self.config["LANGFUSE_BASE_URL"] = "http://langfuse-web:3000"
+            self.console.print("[green][SUCCESS][/green] LangFuse auto-configured from wizard")
+            self.console.print(f"[blue][INFO][/blue] Host: http://langfuse-web:3000")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)